package.json

```diff
@@ -57,7 +57,7 @@
     "@types/lodash.get": "^4.4.7",
     "@types/micromatch": "^4.0.2",
     "@types/node": "^20.3.3",
-    "@types/prettier": "^2.7.2",
+    "@types/prettier": "^3.0.0",
```

Comment:
We don't need the `@types/prettier` https://prettier.io/blog/2023/07/05/3.0.0.html#api-2